j5/backends/console/console.py

```diff
@@ -8,11 +8,14 @@
 class Console:
     """A helper class for console backends."""
 
+    DEFAULT_PRINT: Callable = print  # type: ignore  # noqa: T002
```

Comment:
Nit: Should these class variables be preceded by an underscore to indicate we don't expect anyone to use them outside of maybe tests?

Reply:
Probably, yeah.

I'm planning a refactor of this class anyway, similar to #522 as it could definitely be more pythonic.